### PR TITLE
Feat: created expense list page with faker

### PR DESCRIPTION
**Pull request by Layhout-WB**
## What type of PR is this? (check all applicable)

- [x] Feature

## Screenshots, Recordings

image

Screenshot 2024-12-05 at 5 41 58 in the afternoon

### CODE REVIEW

File: app/app/expenses/_components/ExpenseGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  loading: boolean
+}>) {
+  return (
+    <div className="grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4">
+      {loading ? Array.from({ length: 3 }).map((_, i) => <ExpenseCardPlaceholder key={i} />) : children}
+    </div>
+  )
+}
+
+export function ExpenseCard({
```
Comment (hing-sokheng) [inline app/app/expenses/_components/ExpenseGrid.tsx:24]: I think we should have separate file for each component.

File: app/app/groups/_components/GroupTable.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar'
+import AvatarStack from '@/components/AvatarStack'
+import { Avatar, AvatarFallback } from '@/components/ui/avatar'
 import { Button } from '@/components/ui/button'
-import { cn } from '@/lib/utils'
-import { GroupMember } from '@/services/group.model'
+import { Group } from '@/services/group.model'
 import { LuMoreHorizontal } from 'react-icons/lu'
```
Comment (hing-sokheng) [inline app/app/groups/_components/GroupTable.tsx:5]: LuMoveHorizontal or LuMoreHorizontal?

File: app/app/layout.tsx [hunk 3]
```diff
@@ -7,37 +7,40 @@ import DesktopNav from './_components/DesktopNav'
...
 import { useEffect, useState } from 'react'
-import InitialLoader from '@/components/InitialLoader'
+import Splash from './_components/Splash'
 
 export default function AppLayout({ children }: { children: React.ReactNode }) {
   const pathname = usePathname()
   const { isCollapsed, setIsCollapsed, appLinks, userLoaded, user } = useAppLayout()
 
-  const [isServer, setIsServer] = useState(false)
+  const [isServer, setIsServer] = useState(true)
```
Comment (hing-sokheng) [inline app/app/layout.tsx:16]: should we name it to isLoading or isServerLoading instead?

**Review by hing-sokheng — CHANGES_REQUESTED** [review]

File: app/app/groups/_components/GroupTable.tsx (hunk 2 above)
Comment (Layhout-WB) [reply app/app/groups/_components/GroupTable.tsx:5]: LuMoreHorizontal

**Pull request merged**